### PR TITLE
Adding PHYS213/214

### PR DESCRIPTION
**Pull request by egwhite3**
I really didn't know what to write about these courses' content, since they are mostly just courses you have to take because we're told to. Any suggestions for added content are welcome.

### CODE REVIEW

File: docs/Course Wiki/PHYS Course Offerings/PHYS213.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+In the past, this course has been taught by Professors Wagner and Covey.
+
+## Course Tips
+
+The key to this course is the equation sheet. Learn how to use all the equations and the homework/exams should be fairly simple. There are also provided practice exams released prior to the real thing. They are usually very similar to the actual exam, so they are the best tool to study.
+
+## Life After
+
+Graduation
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS213.md]: Change to something like "The introduction of thermal physics will prepare for students for future courses in semiconductor devices and solid-state physics, including [ECE340 - Semiconductor Electronics](../ECE%20Course%20Offerings/ECE340.md).  Students who wish to go more in depth into thermal physics should take the undergraduate course in statistical mechanics, [PHYS427 - Thermal and Statistical Physics](PHYS427.md)."

File: docs/Course Wiki/PHYS Course Offerings/PHYS213.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# PHYS213
+
+PHYS213 (Thermal Physics) is a 2-credit hour, 8-week long, course that is required for physics majors. It is required for ECE majors. It is offered in the fall, spring, and summer semesters.
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS213.md]: change to "(University Physics: Thermal Physics)", also change to "that is required for ECE majors as a part of the Foundational Mathematics and Science requirement.  It is also required for physics majors."  Also mention it is a second 8-weeks course.

File: docs/Course Wiki/PHYS Course Offerings/PHYS213.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+## Prerequisites
+
+- [PHYS211](PHYS211.md)
+- [MATH241](../MATH%20Course%20Offerings/MATH241.md) (Corequisite)
+
+The official prerequisites for PHYS213 are [PHYS211](PHYS211.md) and credit or concurrent registration in [MATH241](../MATH%20Course%20Offerings/MATH241.md). There is almost no direct overlap in content between the physics courses, and very little calculus, in general, is used.
+
+## When to Take It
+
+ECE majors typically take PHYS213 and [PHYS214](PHYS214.md) during the same semester after taking [PHYS212](PHYS212.md). The two courses are designed to be taken in the same semester, so it ends up filling roughly the same amount of time as a normal 4 credit hour, with the content changing halfway through.
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS213.md]: mention that PHYS213 and 214 are during different halves of the semester

File: docs/Course Wiki/PHYS Course Offerings/PHYS213.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Graduation
+
+## Infamous Topics
+
+- Entropy: It is a very strange concept, so watching the prelecture a few times is recommended.
+
+## Resources
+
+Course website
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS213.md]: Link website

File: docs/Course Wiki/PHYS Course Offerings/PHYS214.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+# PHYS214
+
+PHYS214 (Quantum Physics) is a 2-credit hour, 8-week long, course that is required for physics majors. It is required for ECE majors. It is offered in the fall, spring, and summer semesters.
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS214.md]: change to "(University Physics: Quantum Physics)", also change to "that is required for ECE majors as a part of the Foundational Mathematics and Science requirement. It is also required for physics majors."  Also mention it is a first 8-weeks course.

File: docs/Course Wiki/PHYS Course Offerings/PHYS214.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+## Prerequisites
+
+- [PHYS212](PHYS212.md)
+
+The official prerequisite for PHYS214 is [PHYS212](PHYS212.md), although there is not a large overlap in content.
+
+## When to Take It
+
+ECE majors typically take PHYS213 and [PHYS214](PHYS214.md) during the same semester after taking [PHYS212](PHYS212.md). The two courses are designed to be taken in the same semester, so it ends up filling roughly the same amount of time as a normal 4 credit hour, with the content changing halfway through.
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS214.md]: mention that PHYS213 and 214 are during different halves of the semester

File: docs/Course Wiki/PHYS Course Offerings/PHYS214.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+The introduction to waves and the weird world of quantum physics leads into EE courses such as [ECE329](../ECE%20Course%20Offerings/ECE329.md) and [ECE340](../ECE%20Course%20Offerings/ECE340.md). Students wishing to go more in-depth into quamtum phsyics should look into the dedicated physics courses, [PHYS486](PHYS486.md) and [PHYS487](PHYS487.md).
+
+## Infamous Topics
+
+- Wave Particle Duality: If you have never been introduced to this it is very wacky. Just hold on tight!
+
+## Resources
+
+Course website
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS214.md]: Link course website

**Review by aidanrodgers — CHANGES_REQUESTED** [review]
Some minor changes

File: docs/Course Wiki/PHYS Course Offerings/PHYS214.md (hunk 7 above)
Comment (egwhite3) [reply docs/Course Wiki/PHYS Course Offerings/PHYS214.md]: Website changes every semester...

File: docs/Course Wiki/PHYS Course Offerings/PHYS213.md (hunk 4 above)
Comment (egwhite3) [reply docs/Course Wiki/PHYS Course Offerings/PHYS213.md]: The website changes every semester...

File: docs/Course Wiki/PHYS Course Offerings/PHYS213.md (hunk 1 above)
Comment (egwhite3) [reply docs/Course Wiki/PHYS Course Offerings/PHYS213.md]: IMO it really does nothing for 340 or SS physics in general. I'll put the part about going more in depth

File: docs/Course Wiki/PHYS Course Offerings/PHYS213.md (hunk 4 above)
Comment (aidanrodgers) [reply docs/Course Wiki/PHYS Course Offerings/PHYS213.md]: Sorry, remove course website for both then

File: docs/Course Wiki/PHYS Course Offerings/PHYS213.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # PHYS213
 
-PHYS213 (Thermal Physics) is a 2-credit hour, 8-week long, course that is required for physics majors. It is required for ECE majors. It is offered in the fall, spring, and summer semesters.
+PHYS213 (University Physics: Thermal Physics) is a 2-credit hour, second half 8-week long, course that is required for ECE majors as a part of the Foundational Mathematics and Science requirement. It is also required for physics majors It is required for ECE majors. It is offered in the fall, spring, and summer semesters.
```
Comment (aidanrodgers) [inline docs/Course Wiki/PHYS Course Offerings/PHYS213.md]: remove second "It is required for ECE majors"

**Review by aidanrodgers — CHANGES_REQUESTED** [review]
Just a minor thing

File: docs/Course Wiki/PHYS Course Offerings/PHYS214.md (hunk 7 above)
Comment (rzig) [reply docs/Course Wiki/PHYS Course Offerings/PHYS214.md]: +1, let's not include course website

**Review by aidanrodgers — APPROVED** [review]
Looks good, I will have Lucas look at these too as he took these courses last semester.

**Pull request merged**